### PR TITLE
HBASE-24196 [Shell] Add rename rsgroup command in hbase shell

### PR DESCRIPTION
**Pull request by Reidddddd**
JIRA link: [HBASE-24196](https://issues.apache.org/jira/browse/HBASE-24196)

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| -0 :warning: |  rubocop  |   0m 10s |  The patch generated 5 new + 45 unchanged - 1 fixed = 50 total (was 46)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 12s |  The patch does not generate ASF License warnings.  |
|  |   |   2m 10s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1551 |
| Optional Tests | dupname asflicense rubocop |
| uname | Linux ef7bda08be70 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bcacc4ce93 |
| rubocop | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/artifact/yetus-general-check/output/diff-patch-rubocop.txt |
| Max. process+thread count | 47 (vs. ulimit of 12500) |
| modules | C: hbase-shell U: hbase-shell |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) rubocop=0.80.0 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 25s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 59s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 40s |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 11s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   6m 45s |  hbase-shell in the patch passed.  |
|  |   |  16m 17s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1551 |
| Optional Tests | javac javadoc unit |
| uname | Linux 27c745a00c73 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bcacc4ce93 |
| Default Java | 1.8.0_232 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/testReport/ |
| Max. process+thread count | 1535 (vs. ulimit of 12500) |
| modules | C: hbase-shell U: hbase-shell |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 51s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 34s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 17s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 13s |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 16s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   7m 18s |  hbase-shell in the patch passed.  |
|  |   |  20m 45s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1551 |
| Optional Tests | javac javadoc unit |
| uname | Linux 2f24f57ab1db 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bcacc4ce93 |
| Default Java | 2020-01-14 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/testReport/ |
| Max. process+thread count | 2392 (vs. ulimit of 12500) |
| modules | C: hbase-shell U: hbase-shell |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-shell/src/test/ruby/shell/rsgroup_shell_test.rb [hunk 1]
```diff
@@ -127,5 +127,17 @@ def setup
...
+    define_test 'Test rsgroup rename' do
+      old_rs_group_name = 'test_group'
+      new_rs_group_name = 'renamed_test_group'
+      table_name = 'test_table'
+
+      @hbase.rsgroup_admin.rename_rsgroup(old_rs_group_name, new_rs_group_name)
+      assert_not_nil(@admin.getRSGroup(new_rs_group_name))
+      assert_equal(1, @admin.getRSGroup(new_rs_group_name).getServers.count)
+      assert_equal(1, @admin.listTablesInRSGroup(new_rs_group_name).count)
+      assert_true(@admin.listTablesInRSGroup(new_rs_group_name).contains(org.apache.hadoop.hbase.TableName.valueOf(table_name)))
```
Comment (HorizonNet) [inline hbase-shell/src/test/ruby/shell/rsgroup_shell_test.rb:140]: Does it make sense to also check if the old rsgroup is gone?

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| -0 :warning: |  rubocop  |   0m 11s |  The patch generated 5 new + 45 unchanged - 1 fixed = 50 total (was 46)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m  8s |  The patch does not generate ASF License warnings.  |
|  |   |   2m  3s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1551 |
| Optional Tests | dupname asflicense rubocop |
| uname | Linux 9f65f4256c82 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bcacc4ce93 |
| rubocop | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/artifact/yetus-general-check/output/diff-patch-rubocop.txt |
| Max. process+thread count | 47 (vs. ulimit of 12500) |
| modules | C: hbase-shell U: hbase-shell |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) rubocop=0.80.0 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 35s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 16s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   3m 20s |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   7m  0s |  hbase-shell in the patch passed.  |
|  |   |  16m  3s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1551 |
| Optional Tests | javac javadoc unit |
| uname | Linux 46de6244bd24 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bcacc4ce93 |
| Default Java | 1.8.0_232 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/testReport/ |
| Max. process+thread count | 2391 (vs. ulimit of 12500) |
| modules | C: hbase-shell U: hbase-shell |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 26s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m 17s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m 24s |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 12s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   6m 48s |  hbase-shell in the patch passed.  |
|  |   |  17m 22s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1551 |
| Optional Tests | javac javadoc unit |
| uname | Linux ab58b9654606 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / bcacc4ce93 |
| Default Java | 2020-01-14 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/testReport/ |
| Max. process+thread count | 1568 (vs. ulimit of 12500) |
| modules | C: hbase-shell U: hbase-shell |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1551/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-shell/src/test/ruby/shell/rsgroup_shell_test.rb (hunk 1 above)
Comment (Reidddddd) [reply hbase-shell/src/test/ruby/shell/rsgroup_shell_test.rb:141]: Add that check in the new commit.

**Pull request merged**